keylime/mba/elchecking/elchecker.py

```diff
@@ -3,17 +3,63 @@
 
 from keylime import config, keylime_logging
 from keylime.failure import Component, Failure
-from keylime.mba.elchecking import tests
+from keylime.mba.elchecking import policies, tests
 
 logger = keylime_logging.init_logging("measured_boot")
 
 
-def evaluate_bootlog(
+def policy_load(policy_path: Optional[str] = None) -> str:
```

Comment:
This was the initial behavior of policy_load. Load from an argument or else load from location specified in [tenant]/mb_refstate. I didn't change this because of backward compatibility.